src/Symfony/Component/PropertyInfo/composer.json

```diff
@@ -31,6 +31,7 @@
         "symfony/cache": "^5.4|^6.0",
         "symfony/dependency-injection": "^5.4|^6.0",
         "phpdocumentor/reflection-docblock": "^5.2",
+        "phpdocumentor/type-resolver": "^1.5.1",
```

Comment:
Please update the conflict rule instead. Note that the root composer.json file has to be updated as well.

Reply:
Done!